extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -28,22 +32,34 @@ class AmpAccordion extends AMP.BaseElement {
 
   /** @override */
   buildCallback() {
+    /** @const @private {boolean} */
+    this.isExperimentOn_ = isExperimentOn(this.getWin(), TAG);
```

Comment:
This is a specific experiment.
I missed previous discussion, but I don't think we need a feature flag for this.